sdk/dotnet/Pulumi.Kubernetes.csproj

```diff
@@ -44,6 +44,9 @@
     <PackageReference Include="Pulumi" Version="3.*" />
   </ItemGroup>
 
+  <ItemGroup>
+  </ItemGroup>
```

Comment:
Do we know why this happens? Probably not a big deal, but doesn't look tidy.

Reply:
I actually left the same comment on some other PR (maybe azure-native?). I'm not sure what's causing it.